packages/angular-query-experimental/src/__tests__/inject-mutation.test.ts

```
@@ -1,17 +1,19 @@
   input,
   provideZonelessChangeDetection,
   signal,
 } from '@angular/core'
 import { TestBed } from '@angular/core/testing'
 import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest'
-import { By } from '@angular/platform-browser'
 import { sleep } from '@tanstack/query-test-utils'
 import { QueryClient, injectMutation, provideTanStackQuery } from '..'
-import { expectSignals, setFixtureSignalInputs } from './test-utils'
+import { expectSignals, registerSignalInput } from './test-utils'
+import { firstValueFrom } from 'rxjs'
```

ESLint

[error] `rxjs` import should occur before import of `..`

(import/order)